app/components/avatar_component.html.erb

```diff
@@ -1 +1 @@
-<%= image_tag @image, alt: "Developer's avatar", data: @data, class: "object-cover h-24 w-24 bg-gray-300 rounded-full ring-4 ring-white sm:h-32 sm:w-32" %>
+<%= image_tag image, alt: "Developer's avatar", data: data, class: "object-cover h-24 w-24 bg-gray-300 rounded-full ring-4 ring-white sm:h-32 sm:w-32" %>
```

Comment:
@joemasilotti 

I'm cool with changing ivars to methods, and it looks like you are too since you put in the work! 😄  I don't remember doing this when I set it up in my app, so perhaps I was just a bit more aggressive ignoring rules.

It does keep the code that much more consistent. Net positive there in my book. 👍

Reply:
Agreed! I ran the linter with `--enable-all-linters` so see how much would needed to be done. Then, I slowly added each relevant check to the config file. One that I'd love to add is the linter that checks for missing translations! But I need to merge #110 first.

This particular fix is from the `PartialInstanceVariable` linter.

Reply:
Ah, makes sense. Thanks for sharing the `PartialInstanceVariable` tip so I don't have to search for it. 🙏